site/app/templates/autograding/AutoResults.twig

```diff
@@ -69,14 +69,22 @@
                         {% endif %}
                     </div>
                 {% endif %}
-            {% elseif has_badges %}
-                <div class="no-badge"></div>
-            {% endif %}
-            {# /Badge #}
-
-            <h4>
```

Comment:
This <h4> is removed without being replaced... so now there's just a </h4> floating in the wild